### PR TITLE
Add French language to dataset 🇫🇷 

### PR DESCRIPTION
**Pull request by nSimonFR**
I have added some [Librivox](https://librivox.org/) recording to [speech.csv](speech.csv) in order to support the French language for the dataset. 
It's about ~650 minutes of audio, or about the same as the English dataset.

I have not included mirrors link pointing to another provider - so the recordings are being fetched directly from [archive.org](https://archive.org/), but I modified the fetch script so it prioritizes mirrors (As they are faster to download for me).

I also added a simple script for people who want to know the average recording length of the dataset for each language ([dataset-size.py](dataset-size.py)).

The main project -[ tomasz-oponowicz/spoken_language_identification](https://github.com/tomasz-oponowicz/spoken_language_identification) - might need to be tweaked in order to support the new language.

🇫🇷  🌟 🌟 🇫🇷

### CODE REVIEW

**Comment by tomasz-oponowicz** [comment]
@nSimonFR Absolutely awesome job :) +1 I will review and merge it soon.

File: speech.csv [hunk 1]
```diff
@@ -1,4 +1,33 @@
 Group,Language,Sex,Extension,Reader,Duration,Url,Mirror
+train,fr,female,mp3,Jc Guan,00:43:54,https://ia800203.us.archive.org/16/items/comte_monte_cristo_jg_librivox/montecristo_036_dumas_64kb.mp3,
+train,fr,male,mp3,Ariodante,00:55:23,http://www.archive.org/download/princesse_monpensier_ar_librivox/monpensier_lafayette_64kb.mp3,
+train,fr,female,mp3,Nadine Eckert-Boulet,00:23:21,https://ia800207.us.archive.org/18/items/fauteuilhante_0911_librivox/fauteuilhante_17_leroux_64kb.mp3,
+train,fr,male,mp3,Bernard,00:22:03,https://ia601201.us.archive.org/6/items/lettrespersanest1_1607_librivox/lettrespersanes1_69_montesquieu_64kb.mp3,
+train,fr,female,mp3,Ezwa,00:22:11,http://ia801403.us.archive.org/9/items/nousdeux_0807_ezwa_librivox/nousdeux_2_bilhaud_64kb.mp3,
+train,fr,male,mp3,Gilles G. Le Blanc,00:22:37,http://www.archive.org/download/lupincontresholmes_1311_librivox/lupincontresholmes_08_leblanc_64kb.mp3,
+train,fr,female,mp3,Anne,00:05:27,http://www.archive.org/download/delespritdeslois_anlo_1004_librivox/delespritdeslois_05_08_montesquieu_64kb.mp3,
```
Comment (tomasz-oponowicz) [inline speech.csv]: The training track has to be at least 5m 30s long. Is it possible to find a replacement?

File: speech.csv [hunk 2]
```diff
@@ -1,4 +1,33 @@
...
+train,fr,male,mp3,Herman Roskams,00:09:00,http://www.archive.org/download/compilationpoemes_004_1305-2_librivox/poemes004_artparlersansriendire_hr_64kb.mp3,
+train,fr,female,mp3,Nathalie Mussard,00:18:52,http://www.archive.org/download/miserables_t3_1108_librivox/lesmiserables_t3_33_hugo_64kb.mp3,
+train,fr,male,mp3,jeremlerigolo,00:12:52,http://www.archive.org/download/la_commune_1312_librivox/commune_07_michel_64kb.mp3,
+train,fr,female,mp3,Kaviraf,00:09:38,http://www.archive.org/download/miserables_t3_1108_librivox/lesmiserables_t3_42_hugo_64kb.mp3,
+train,fr,male,mp3,Sergio Baldelli,00:13:02,http://www.archive.org/download/la_commune_1312_librivox/commune_02_michel_64kb.mp3,
+train,fr,female,mp3,Saab,00:24:56,http://ia600702.us.archive.org/4/items/miserables_t3_1108_librivox/lesmiserables_t3_75_hugo_64kb.mp3,
+train,fr,male,mp3,Enkerli,00:07:36,http://www.archive.org/download/bouledesuif_0904_librivox/bouledesuif_12_maupassant_64kb.mp3,
+train,fr,female,mp3,Claire Dyard,00:16:45,http://www.archive.org/download/le_pays_des_fourrures_0712_librivox/pays_fourrures_06_verne_64kb.mp3,
```
Comment (tomasz-oponowicz) [inline speech.csv]: In order to keep all languages equally balanced we need 28 training tracks (14 males and 14 females). There are 27 training tracks now, i.e. one male track is missing.

File: speech.csv [hunk 3]
```diff
@@ -1,4 +1,33 @@
 Group,Language,Sex,Extension,Reader,Duration,Url,Mirror
+train,fr,female,mp3,Jc Guan,00:43:54,https://ia800203.us.archive.org/16/items/comte_monte_cristo_jg_librivox/montecristo_036_dumas_64kb.mp3,
```
Comment (tomasz-oponowicz) [inline speech.csv]: Can you replace `ia800203.us.archive.org` with `www.archive.org`? From my experience `*.archive.org` servers are temporary and may go down.

**Review by tomasz-oponowicz — CHANGES_REQUESTED** [review]
Looks good to me.

I found few minor things. Please take a look at my comments.

**Comment by nSimonFR** [comment]
Hello Tomasz,
I have made the necessary changes, let met know if something still isn't right !

**Review by tomasz-oponowicz — APPROVED** [review]

**Pull request merged**